server/webdriver/tests/standalone/sanity.py

```diff
@@ -0,0 +1,338 @@
+    self.driver.quit()
+
+  def add_result(self, result: Result) -> None:
+    print(f"#{len(self.results) + 1}", result)
+    self.csv_writer.writerow(result.__dict__)
+    self.file.flush()
+    self.results.append(result)
+
+  def home(self, page: WebPage):
+    self.driver.get(page.url)
+
+    page.title = self.driver.title if page.title is None else page.title
+
+    # topic cards
+    topic_cards = find_elems(self.driver, By.CLASS_NAME, "topic-card")
+    if topic_cards is None or len(topic_cards) == 0:
+      self.add_result(fresult(page, "No topic cards."))
+      return
+
+    explore_landing_pages = []
+    for topic_card in topic_cards:
+      topic_title_elem = find_elem(topic_card, By.CLASS_NAME,
+                                   "topic-card-title")
+      if topic_title_elem is None:
+        self.add_result(
+            fresult(
+                page,
+                "No explore landing title on one of the cards.",
+            ))
+        return
+
+      topic_url_elem = find_elem(topic_card, By.TAG_NAME, "a")
+      if topic_url_elem is None:
+        self.add_result(
+            fresult(
+                page,
+                "No explore landing URL on one of the cards.",
+            ))
+        return
+
+      explore_landing_pages.append(
+          WebPage(
+              PageType.EXPLORE_LANDING,
+              topic_title_elem.text,
+              topic_url_elem.get_attribute("href"),
+          ))
+
+    # Pass
+    self.add_result(presult(page))
+
+    for explore_landing_page in explore_landing_pages:
+      self.explore_landing(explore_landing_page)
+
+  def explore_landing(self, page: WebPage):
+    self.driver.get(page.url)
+
+    page.title = self.driver.title if page.title is None else page.title
+
+    # topics
+    topics = find_elems(self.driver, By.CLASS_NAME, "item-list-text")
+    if topics is None or len(topics) == 0:
+      self.add_result(fresult(page, "No topics."))
+      return
+
+    # queries
+    queries_parent = find_elem(
+        self.driver,
+        By.CSS_SELECTOR,
+        "#dc-explore-landing > div > div > div.topic-container > div.topic-queries",
+    )
+    if queries_parent is None:
+      self.add_result(fresult(page, "No queries."))
+      return
+    queries = find_elems(queries_parent, By.TAG_NAME, "a")
+    if queries is None or len(queries) == 0:
+      self.add_result(fresult(page, "No queries."))
+      return
+
+    # Pass
+    self.add_result(presult(page))
+
+    explore_links = topics + queries
+    explore_pages = []
+    for link in explore_links:
+      explore_pages.append(
+          WebPage(PageType.EXPLORE, link.text, link.get_attribute("href")))
+    for explore_page in explore_pages:
+      self.explore(explore_page, True)
+
+  def explore(self, page: WebPage, recurse: bool = False):
+    self.driver.get(page.url)
+
+    page.title = self.driver.title if page.title is None else page.title
+
+    # Wait 5 secs for charts container to load
```

Comment:
This function https://github.com/datacommonsorg/website/blob/master/server/webdriver/shared.py#L56 should help on the timeout issue. It checks all the async element to fully load